Equilobe.DailyReport.Models/ReportPolicy/User.cs

```diff
@@ -10,6 +10,7 @@ namespace Equilobe.DailyReport.Models.ReportPolicy
     public class User
     {
         public string JiraUserKey { get; set; }
+        public string JiraDisplayName { get; set; }
```

Comment:
@vladichimescu 
Proprietatea astea nu are ce sa caute pe clasa asta, chiar daca tu ai nevoie de ea in UI. 

O sa ai nevoie de o clasa noua.